### PR TITLE
Introduce risks controller with risks to self form

### PR DESCRIPTION
**Pull request by cesidio**

### CODE REVIEW

File: app/controllers/concerns/wizardable_controller.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
+module WizardableController
```
Comment (whoojemaflip) [inline app/controllers/concerns/wizardable_controller.rb]: since this isn't a controller, should it be called x_controller?

File: app/views/healthcare/_allergies.html.slim [hunk 2]
```diff
@@ -1,4 +1 @@
-= form_for form, url: form_path, method: :put do |f|
-  = f.radio_toggle_with_textarea :allergies
-
-  = render partial: 'buttons', locals: { f: f }
+= f.radio_toggle_with_textarea :allergies
```
Comment (whoojemaflip) [inline app/views/healthcare/_allergies.html.slim:1]: why does this PR affect the healthcare partials?

Comment (cesidio) [inline app/views/healthcare/_allergies.html.slim:1]: Should have been done in a different PR, but I removed an helper method from the controller and made the healthcare views more consistent with the ones we have in risks
